resqpy/well.py

```diff
@@ -865,7 +866,7 @@ def load_from_xml(self, node, hdf5_source_model = None):
       md_datum_part = relatives_model.part_for_uuid(md_datum_uuid)
       assert md_datum_part, 'md datum part not found in model'
       self.md_datum = MdDatum(self.model, md_datum_root = relatives_model.root_for_part(md_datum_part))
-      ds_uuid = bu.uuid_from_string(rqet.find_tag(rqet.find_tag(node, 'DeviationSurvey'), 'UUID'))
+      ds_uuid = bu.uuid_from_string(rqet.find_tag_text(rqet.find_tag(node, 'DeviationSurvey'), 'UUID'))
```

Comment:
I saw Andy wrote some functions for nested tag lookup, could perhaps be useful?

```suggestion
      ds_uuid = bu.uuid_from_string(rqet.find_nested_tags_text(node, ['DeviationSurvey', 'UUID']))
```